src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+
```

Comment:
[Common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/oop)
**Please don’t add redundant empty lines to your code.**